.github/dependabot.yml

```diff
@@ -8,9 +8,7 @@ updates:
       interval: 'daily'
     labels:
       - "Dependencies"
```

Comment:
Based on [the documentation](https://docs.github.com/en/code-security/supply-chain-security/keeping-your-dependencies-updated-automatically/configuration-options-for-dependency-updates#labels) I'm reading, "Dependencies" isn't needed here since "dependencies" is the default label for all pull requests.

> By default, Dependabot raises all pull requests with the dependencies label.